docs/en/ingest-management/elastic-agent/run-elastic-agent-standalone.asciidoc

```diff
@@ -41,7 +41,7 @@ image::images/kibana-fleet-policies-default-yaml.png[{fleet} showing default age
 NOTE: The policy generated by {fleet} already contains the correct {es} address
 and port for your setup. If you run everything locally, the address is
 `127.0.0.1:9200`. If you use our
-https://www.elastic.co/cloud/elasticsearch-service[hosted {ess}] on {ecloud},
+{ess-product}[hosted {ess}] on {ecloud},
```

Comment:
we may need to discuss how to phrase this and what to cite about Fleet Server at this point and cloud.  The stand-alone Agent can still connect direct to ES and does not need Fleet or Fleet-Server for that matter.  The citation of Fleet at all was because Kibana basically set up Fleet for you and it was easy to use it to provide a way to get the ES creds needed.  We may want to generalize this content to just say 'put in your ES user and creds into your config to use stand-alone Agent'.

Reply:
Yup! I'll make changes for standalone in a different PR. I think our docs for running standalone won't be great until we get all the config settings documented (there are multiple stalled PRs for that work and lots of unresolved questions).